Listenbrainz/ViewModel/ShazamViewModel.swift

```diff
@@ -38,12 +38,18 @@ final class ShazamViewModel: NSObject, ObservableObject {
     private lazy var audioEngine: AVAudioEngine = .init()
     private lazy var inputNode = self.audioEngine.inputNode
     private lazy var bus: AVAudioNodeBus = 0
+    @Published private(set) var isMatching = false
+    private let audioFileURL = Bundle.main.url(forResource: "ird", withExtension: "mp3")
```

Comment:
@gauravbhardwaj7 we want to detect the mp3 files already present in the device of the user and then have them select which song they wanna match.

Reply:
We can also allow the user to input(add) files and then save that file in the storage of our app and match then

Reply:
Alright!, will implement it:)

Reply:
implemented it, please have a look :)